### PR TITLE
5 non imported uses dont get considered as dependencies

### PR DESCRIPTION
**Pull request by ta-tikoma**

### CODE REVIEW

File: src/Asserts/Dependencies/ObjectDependenciesDescription.php [hunk 1]
```diff
@@ -30,15 +30,19 @@ public static function make(string $path): ?self
...
-        $useUses = ServiceContainer::$nodeFinder->findInstanceOf($description->stmts, Node\Stmt\UseUse::class);
+        /** @var Node\Name [] $names */
+        $names = ServiceContainer::$nodeFinder->findInstanceOf($description->stmts, Node\Name::class);
 
         $description->uses = new ObjectUses(
             array_map(
-                static function (Node\Stmt\UseUse $useUse): string {
-                    return $useUse->name->toCodeString();
+                static function (Node\Name $nodeName): string {
+                    $name = $nodeName->toCodeString();
```
Comment (nunomaduro) [inline src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: Seems that stuff such as `parent`, or `true` or being captured here.

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: src/Asserts/Dependencies/ObjectDependenciesDescription.php (hunk 1 above)
Comment (ta-tikoma) [reply src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: and native function names, but I think it is dependents too

Comment (nunomaduro) [reply src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: Do you plan to work on this soon? Or should I try to address this?

Comment (ta-tikoma) [reply src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: I hope this doesn't have a negative effect

Comment (nunomaduro) [reply src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: Well, at least things like `true` or `null`, can't be considered as "dependents". That's basically native PHP code.

Comment (nunomaduro) [reply src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: @ta-tikoma  You think you can address the comment above soon, or you wish that I complete this pull request myself?

Comment (ta-tikoma) [reply src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: @nunomaduro sorry for long time. make your pull request, please

Comment (nunomaduro) [reply src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: @ta-tikoma If you wish - you can also add me as maintainer of this package, and we can maintainer together.

Comment (nunomaduro) [reply src/Asserts/Dependencies/ObjectDependenciesDescription.php:39]: Perfect. I will address this pull request after the 15 February. I am busy until then.

**Pull request merged**